lightsaber/gradle-plugin/src/main/java/com/joom/lightsaber/plugin/JavaLightsaberPluginExtension.kt

```diff
@@ -19,5 +19,6 @@ package com.joom.lightsaber.plugin
 open class JavaLightsaberPluginExtension {
   var processTest: Boolean = true
   var validateUsage: Boolean = true
+  var validateUnusedImports: Boolean = false
```

Comment:
Why isn't it `true` by default like `validateUsage`?

Reply:
I'm planning on making a separate PR that will:
- Enable `validateUnusedImports` by default
- Make it possible to change the default value for all modules via single `gradle.properties` property
- Add an explanation to an error message on how to disable the validation

Enabling `validateUnusedImports` by default will break existing projects and currently it's really hard to disable it in a multi-module environment, that's why I want it to be disabled by default for now.